airbyte-integrations/connectors/source-greenhouse/acceptance-test-config.yml

```diff
@@ -17,6 +17,19 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
+      empty_streams:
+        [
+          "applications.demographics_answers",
+          "applications.interviews",
+          "demographics_answer_options",
+          "demographics_answers",
+          "demographics_answers.answer_options",
+          "demographics_question_sets",
+          "demographics_question_sets.questions",
+          "demographics_questions",
+          "interviews",
+          "rejection_reasons",
```

Comment:
Is there a way to fill those streams? If yes, then maybe we could create a ticket for it?

Reply:
^